game/src/main/scala/io/aigar/game/Grid.scala

```diff
@@ -0,0 +1,22 @@
+package io.aigar.game
+
+import com.github.jpbetz.subspace._
+import java.util.Random
+
+object Grid {
+  final val WidthPerPlayer = 100
+  final val HeightPerPlayer = 100
+}
```

Comment:
Smart idea to create a relation between the grid size and the number of players.